rainbow-me/rainbow
# Add images to iOS notifications

Fixes TEAM2-71

## What changed (plus any additional context for devs)

This PR adds images to iOS notifications.

## Screen recordings / screenshots

<img src="https://user-images.githubusercontent.com/7336481/179640176-f1a1f54a-3f9a-49f6-bc17-d3a07b08a2a4.PNG" width="300px"></img>

## What to test

Can test on TestFlight 1.6.17 (9)

- Ensure notifications still work properly in general
- Ensure images are attached to notifications that include images

### How to test notifications w/ images

1. Make sure you are subscribing to a topic via Settings -> Notifications Debug in the Rainbow App
2. Head to the [Firebase Notifications Composer](https://console.firebase.google.com/u/0/project/_/notification)
3. Create a new notification
4. Add a mock title, description and image
5. **IMPORTANT**: make sure you target a **Topic**
6. Insert `owner_1_{your address here}_received` into the input field
7. Send the notification off!


## Final checklist
